.github/workflows/sql-workbench-test-and-build-workflow.yml

```diff
@@ -51,7 +58,7 @@ jobs:
           yarn test:jest --coverage
 
       - name: Upload coverage
-        if: always()
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
@Yury-Fridlyand does this need to be ${{always() && matrix.os == 'ubuntu-latest' }}??

Reply:
D'oh, right! It should be on `2.x` as well... 
Late catch better than no catch.